css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  text-align: center;
+  width: 1200px;
+  font-family: 'DM Sans', sans-serif;
+}
+
+img {
+  height: 250px;
+  width: 200px;
+}
+
+#wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.container1 {
+  width: 20%;
+}
+
+.size {
+  color: gray;
+  text-decoration: line-through;
+}
+
+.size-top {
+  color: gray;
+}
+
+.dollar {
+  background-color: rgb(253, 238, 240);
+  font-weight: bold;
+  font-size: 12px;
+}
+
+.dress2 {
+  padding-right: 4rem;
```

Comment:
What is all of the padding for the remaining CSS rules here doing for you? It is a lot of padding, and there is probably a better way 🤔